python/dask_cudf/dask_cudf/io/tests/test_json.py

```diff
@@ -97,3 +97,30 @@ def test_read_json_nested(tmp_path):
         # Ensure not passing kwargs also reads the file.
         actual = dask_cudf.read_json(f)
         dd.assert_eq(actual, actual_pd)
+
+
+def test_read_json_aggregate_files(tmp_path):
+    df1 = dask.datasets.timeseries(
+        dtypes={"x": int, "y": int}, freq="120s"
+    ).reset_index(drop=True)
+    json_path = str(tmp_path / "data-*.json")
+    df1.to_json(json_path)
+
+    df2 = dask_cudf.read_json(json_path, aggregate_files=2)
+    assert df2.npartitions * 2 == df1.npartitions
```

Comment:
Any chance df1 could lead to an odd number of files and thus aggregating two-by-two would lead to a different number of partitions, e.g., `3 * 2 == 5` (number of files is `5`)?

Reply:
You're right - We never know if/when `dask.datasets.timeseries` behavior will change. I can generalize the assertion to:

```python
assert df2.npartitions == math.ceil(df1.npartitions / 2)
```